### ISSUE TITLE
Question regarding jar for external nodules vs hul-modules 

### ISSUE THREAD

**Issue by asciim0** [openpreserve/jhove#641]
Quick question regarding JHOVE modules. There are two different places where they are stored within the JHOVE architecture – the original hul modules are all in /jhove/jhove-modules/ while PNG. WARC and epub are in a different place altogether. 
Furthermore – and this is that part that causes us a bit of trouble – module-independent jars exist for the original hul modules but not for the jhove-ext-modules, when downloading the pre-compiled JHOVE, they are all in one shared jar (jhove-ext-modules-1.24.2.jar).  We are wondering what the reason for this is and if this could be changed?

Within our infrastructure (Rosetta), we are leveraging JHOVE on a file format group / module basis – it would be helpful if all modules are packaged the same way, i.e., one distinct jar per module and not several packaged in one. Is this something that would be possible in the future? I know that we could most likely compile atomic modules ourselves … but apart from potential developer overhead we would like to have one authoritative jar, delivered through the OPF general release, which all customers can use alike in system integrated and stand-alone manner. Hope this makes sense.

**Comment by asciim0** [comment]
Is there any idea when this will be implemented?

**Comment by asciim0** [comment]
Asking again whether there are any plans here to change this or if there is a reason for all the jhove-ext-modules to be in one jar?

**Comment by samalloing** [comment]
Yes, this is also something we are wondering. Seems like this is redundant.

Sam
cc @carlwilson

**Comment by carlwilson** [comment]
We will be separating the external modules in the next JHOVE release.

**Comment by carlwilson** [comment]
While this had to miss the 1.26 release in order for us to get it out in a reasonable time scale the project structure is the first set of changes we're making for 1.27 development.

**Comment by asciim0** [comment]
This didn't seem to make it into the 1.28 RC either, correct? Any idea when this will be expected?

**Comment by asciim0** [comment]
Any updates on when this will be delivered? It's become good practice to ask about this at least twice a year ;-P

**Comment by asciim0** [comment]
Since it's about that time again (and I have a quarterly call I'm reporting this on) ... any news when this will be delivered?
